pyfixest/utils.py

```diff
@@ -9,14 +9,18 @@ def ssc(adj=True, fixef_k="none", cluster_adj=True, cluster_df="min"):
 
     Parameters
     ----------
-        adj: bool, default True
-            If True, applies a small sample correction of (N-1) / (N-k) where N is the number of observations
-            and k is the number of estimated coefficients excluding any fixed effects projected out in either fixest::feols() or lfe::felm().
-        fixef_k: str, default "none"
-            Equal to 'none': the fixed effects parameters are discarded when calculating k in (N-1) / (N-k).
-        cluster_adj: bool, default True
-            If True, a cluster correction G/(G-1) is performed, with G the number of clusters.
-        cluster_df: str, default "conventional"
+        adj : bool, default True
+            If True, applies a small sample correction of (N-1) / (N-k) where N
+            is the number of observations and k is the number of estimated
+            coefficients excluding any fixed effects projected out in either
```

Comment:
Here I seem to have copied the docs of one of my R packages, `fwildclusterboot`...